apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -402,4 +424,34 @@ export class InboxController {
       })
     );
   }
+
+  @KeylessAccessible()
+  @ExternalApiAccessible()
```

Comment:
I wonder, should we only want to keep it as keyless acessible and hide API access?